class/appcat.yml

```diff
@@ -0,0 +1,21 @@
+parameters:
+  kapitan:
+    compile:
```

Comment:
Is there a reason we don't add all dependencies right away? such as `crossplane.libsonnet` that is needed for providers?

Reply:
Yes, because that's how commodore handles library dependencies. We shouldn't add them explicitly as they'll be present if the corresponding component is installed on the cluster. We only need to add the dependencies in the tests as there this doesn't work.

Jsonnet library dependencies in commodore aren't as good as they could be yet, so for now we're stuck with it.